src/coprocessor/dag/executor/index_scan.rs

```diff
@@ -145,6 +148,7 @@ impl IndexScanExecutor {
 
 impl Executor for IndexScanExecutor {
     fn next(&mut self) -> Result<Option<Row>> {
+        self.count += 1;
```

Comment:
You'd better increase count only when meets some value since we will support stream in the future.